chapters/remote/jinad.md

```diff
@@ -16,6 +16,16 @@ We use the tag `latest-daemon` in the below command, which uses the latest relea
 docker pull jinaai/jina:latest-daemon
 ```
 
+##### Deploying AWS EC2 instance inside jinad by [terraform](https://www.terraform.io/)
+
+We extract a module [jinad-aws](https://registry.terraform.io/modules/jina-ai/jinad-aws/jina/latest) to simplify this work. 
+
+With this terraform module We can: 
+- Deploy one or more instance inside jinad
```

Comment:
Be careful of using the same word `instance`. The reader will get confused between EC2 instance and a jina instance.
```suggestion
- Deploy one or more Flows inside jinad
```

Reply:
Cool, that's the point.
Let me make this sentence more clear.